R/GeneCorVSgotermEDA.R

```diff
@@ -0,0 +1,168 @@
+# EDAtool.R
+# Purpose: Exploratory Data Analysis tool project for BCB420 2019
+# Version: 2.0.0
+# Version history: 1.0.0
+# Date: 02 April 2019
+# Author: Yuhan Zhang(yuhan.zhang@mail.utoronto.ca)
+# License:
+#MIT
+#
+# Input: set of genes of interest,could be fetched from a 5-letter system code like PHALY
+# Output: heatmap of co-expression correlation, heatmap of GO semantic similarity of each pair of genes
+# in the set. A scatterplot of correlation vs similarity and a fitted line within confidence level.
+#
+# Dependencies: devtools,ggplot,ggcorrplot, GOSemSim
+#
+
+# ====  FUNCTIONS  =============================================================
+
+
+
+
+# function stub taken from Dr. Steipe's BCB420.2019.ESA package (https://github.com/hyginn/BCB420.2019.ESA)
+corGenes <- function(A, B, prf) {
+  # Calculate pearson correlation between gene expression
+  # profiles A and B in prf identified by the gene symbol.
+  # A and B can be either gene symbol or index.
+
+  r <- cor(prf[A, ], prf[B, ], use = "pairwise.complete.obs")
+  return(r)
+}
+
+
+# ====  PACKAGES  ==============================================================
+# Load all required packages.
+#
+# Use non-standard libraries with  package::function() idiom if possible.
+
+
+#BiocManager::install("GOSemSim", version = "3.8")
+#BiocManager::install("org.Hs.eg.db", version = "3.8")
+#install.packages("ggcorrplot")
+# packages for calculating GO similarity
+
+
+
+
+#'
+#' \code{myPlot()}
```

Comment:
I think you can make the name of your function more expressive (i.e. specify what kind of plots you're making)

Reply:
fixed to myScatterPlot()